<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formcadcursos</class>
 <widget class="QDialog" name="formcadcursos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>498</height>
   </rect>
  </property>
   <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Cadastro de Curso</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nome: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leNome"/>
     </item>
    </layout>
    </widget>
   <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Coordenador:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCoord"/>
     </item>
    </layout>
     </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Carga Horária:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCargaH"/>
     </item>
    </layout>
     </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>301</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Manhã</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Tarde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>Noite</string>
       </property>
      </widget>
     </item>
    </layout>
    </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
     <property name="text">
     <string>Turnos Disponíveis:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
     </property>
    <property name="title">
     <string>Grade Curricular:</string>
    </property>
    <widget class="QScrollArea" name="scrollGrade">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>80</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>279</width>
        <height>78</height>
       </rect>
      </property>
     </widget>
    </widget>
</ui>

